modules/plugin/imagemosaic/src/test/java/org/geotools/gce/imagemosaic/ImageMosaicRATTest.java

```diff
@@ -135,6 +136,65 @@ public void testRATFromGeotiff() throws IOException, FactoryException {
         reader.dispose();
     }
 
+    @Test
+    public void testReloadRAT() throws IOException, FactoryException {
+        // setup data
+        File mosaicSource = TestData.file(this, "bluetopo");
+        String mosaicName = "bluetopoReload";
+        File mosaicDirectory = new File("target", mosaicName);
+        FileUtils.deleteQuietly(mosaicDirectory);
+        FileUtils.copyDirectory(mosaicSource, mosaicDirectory);
+
+        // the aux files
+        String pamFileName = "BlueTopo_BH4JS577_20230918.tiff.aux.xml";
+        File small = new File(mosaicDirectory, "BlueTopo_BH4JS577_20230918.tiff.small.aux.xml");
+        File full = new File(mosaicDirectory, pamFileName);
+
+        // rename p1 so that it won't be found
+        File hideMe = new File(full.getParent(), "hideme.xml");
+        assertTrue(full.renameTo(hideMe));
+        assertTrue(small.renameTo(full));
+
+        // create mosaic and grab the pam databset from resource info
+        ImageMosaicReader reader = getReader(fileToUrl(mosaicDirectory));
+        ResourceInfo info = reader.getInfo(mosaicName);
+        assertThat(info, CoreMatchers.instanceOf(PAMResourceInfo.class));
+        PAMResourceInfo pamInfo = (PAMResourceInfo) info;
+        PAMDataset pam = pamInfo.getPAMDataset();
+
+        // grab the raster attribute table
+        assertNotNull(pam);
+        List<PAMDataset.PAMRasterBand> bands = pam.getPAMRasterBand();
+        PAMDataset.PAMRasterBand b2 = bands.get(2);
+        GDALRasterAttributeTable rat = b2.getGdalRasterAttributeTable();
+
+        List<Row> originalRows = rat.getRow();
+
+        // not add back the PAM file and force reload
```

Comment:
is it "not add" or is it actually "now add" ?

Reply:
Yep